plexapi/utils.py

```diff
@@ -277,3 +277,23 @@ def download(url, filename=None, savepath=None, session=None, chunksize=4024, un
             handle.extractall(savepath)
     # finished; return fillpath
     return fullpath
+
+
+def tag_helper(tag, items, locked=True, remove=False):
+    """Simple tag helper for editing a object."""
+    if not isinstance(items, list):
+        items = [items]
+
+    d = {}
```

Comment:
I'm not sure if there will be any negative effects for not setting the indices back to 0. What happens after you edit the tags with the Python API, then go an edit the tags in Plex Web?